lib/fluent/plugin/in_windows_eventlog.rb

```diff
@@ -55,6 +59,20 @@ def configure(conf)
                           else
                             method(:no_encode_record)
                           end
+      if @store_pos
+        unless system_config.root_dir
+          raise Fluent::ConfigError, "windows_eventlog: 'root_dir' in <system> parameter must be specified with 'store_pos' parameter."
+        end
+        @chs.map {|ch|
+          config = Fluent::Config::Element.new('storage',
+                                               "in_windows_eventlog_pos_#{ch.gsub(' ', '_')}", {
```

Comment:
Storage usage should be short and simple: it should be unique per owner plugins, not globally.
It looks enough to use channel name as usage directly.